app/Handlers/Commands/Metric/AddMetricPointCommandHandler.php

```diff
@@ -36,7 +36,7 @@ public function handle(AddMetricPointCommand $command)
         ];
 
         if ($createdAt) {
-            $data['created_at'] = Carbon::createFromFormat('U', $createdAt)->format('Y-m-d H:i:s');
+            $data['created_at'] = Carbon::createFromFormat('U', $createdAt, config('cachet.timezone'))->format('Y-m-d H:i:s');
```

Comment:
basically, use it in the stuff that we'd never want to separate, or unit test

do you agree?